tasks/libs/common/git.py

```diff
@@ -132,3 +150,35 @@ def check_clean_branch_state(ctx, github, branch):
             ),
             code=1,
         )
+
+
+def get_last_commit(ctx, repo, branch):
```

Comment:
NIT: Maybe adding `repo='datadog-agent'` to show an example (we can wonder is it `datadog-agent` or `DataDog/datadog-agent` ?)

Reply:
I will not change this because I should change the arguments order and I don't want to put `branch` before `repo` (and I don't want to put a default branch as well)
Added a comment